### PR TITLE
Cfy 5223 move elasticsearch curator packaging to omnibus

### PR DESCRIPTION
**Pull request by gilzellner**

### CODE REVIEW

File: packaging/elasticsearch_curator/omnibus/config/projects/elasticsearch-curator.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+name "elasticsearch-curator"
+maintainer "Gigaspaces"
+homepage "http://getcloudify.org/"
+
+override :cacerts, version: '2015.10.28', source: { md5: '782dcde8f5d53b1b9e888fdf113c42b9' }
+override :pip, version: '7.1.2', source: { md5: '3823d2343d9f3aaab21cf9c917710196' }
+override :setuptools, version: '18.5', source: { md5: '533c868f01169a3085177dffe5e768bb' }
+
+install_dir "/opt/es_curator" #TODO: is this real ?
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/config/projects/elasticsearch-curator.rb]: change to /opt/es-curator, also in manager blueprints.

File: packaging/elasticsearch_curator/omnibus/elasticsearch-curator/recipes/install.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+# reload ohai data
+ohai 'reload' do
+  action :reload
+end
+
+# should only be one of each rpm/deb
+pkg_path = Dir.glob("#{node['etc']['passwd']['vagrant']['dir']}/cloudify/pkg/cloudify*rpm")[0] if node['platform_family'] == 'rhel'
+pkg_path = Dir.glob("#{node['etc']['passwd']['vagrant']['dir']}/cloudify/pkg/cloudify*deb")[0] if node['platform_family'] == 'debian'
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/elasticsearch-curator/recipes/install.rb]: remove deb and windows

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+
+PROGNAME=$(basename $0)
+INSTALLER_DIR="/opt/es_curator/"
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst]: /opt/es-curator/

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+PREFIX="/usr"
+
+function error_exit
+{
+  echo "${PROGNAME}: ${1:-"Unknown Error"}" 1>&2
+  exit 1
+}
+
+
+#ln -sf $INSTALLER_DIR/embedded $INSTALLER_DIR/env || error_exit "Cannot link $INSTALLER_DIR/env"
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst]: remove

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst [hunk 5]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+
+PROGNAME=$(basename $0)
+INSTALLER_DIR="/opt/es_curator/"
+PREFIX="/usr"
+
+function error_exit
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst]: remove

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  echo "${PROGNAME}: ${1:-"Unknown Error"}" 1>&2
+  exit 1
+}
+
+
+#ln -sf $INSTALLER_DIR/embedded $INSTALLER_DIR/env || error_exit "Cannot link $INSTALLER_DIR/env"
+
+echo "Thank you for installing elasticsearch curator!"
+
+exit 0
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postinst]: remove

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postrm [hunk 7]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Perform necessary amqp-influxdb removal steps
+# after package is uninstalled.
+#
+
+echo "amqp-influxdb has been uninstalled!"
+
+exit 0
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postrm]: remove

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postrm [hunk 8]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Perform necessary amqp-influxdb removal steps
+# after package is uninstalled.
+#
+
+echo "amqp-influxdb has been uninstalled!"
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/postrm]: es-curator

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/preinst [hunk 9]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# before package is installed.
+#
+
+echo "You're about to install amqp-influxdb!"
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/preinst]: es-curator

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/prerm [hunk 10]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Perform necessary setup steps
+# prior to installing package.
+#
+
+PROGNAME=$(basename $0)
+
+function error_exit
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/prerm]: remove

File: packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/prerm [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+{
+  echo "${PROGNAME}: ${1:-"Unknown Error"}" 1>&2
+  exit 1
+}
+
+exit 0
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/package-scripts/elasticsearch-curator/prerm]: remove

File: packaging/elasticsearch_curator/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+  #   it { should be_directory }
+  #   it { should be_readable }
+  # end
+  #
+  # describe file('/opt/cloudify/bin') do
+  #   it { should be_directory }
+  #   it { should be_readable }
+  # end
+
+  describe file('/opt/amqpinflux/embedded') do
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb]: /opt/es-curator/

File: packaging/elasticsearch_curator/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb [hunk 13]
```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+if ['centos', 'redhat'].include?(os[:family])
+
+  # describe package('cloudify') do
```
Comment (gilzellner) [inline packaging/elasticsearch_curator/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb]: remove all comments

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**